app/forms/transfer/task/conditions_met_task_form.rb

```diff
@@ -0,0 +1,14 @@
+class Transfer::Task::ConditionsMetTaskForm < BaseTaskForm
+  attribute :confirm_all_conditions_met, :boolean
+
+  def initialize(tasks_data, user)
+    @tasks_data = tasks_data
+    @user = user
+    @project = @tasks_data.project
```

Comment:
I don't think `confirm_all_conditions_met` is being initialized here so the ui never updates. The test you have passes, but I suspect if you go back to the task page an assert the box is ticked, it will fail.

This is also the case on Conversions.

Something like:

```
  def initialize(tasks_data, user)
    @tasks_data = tasks_data
    @user = user
    @project = @tasks_data.project
    super(@tasks_data, @user)
    self.confirm_all_conditions_met = @project.all_conditions_met
  end
```

The assignment happens after the super class has been setup.